index.html

```diff
@@ -1,116 +1,229 @@
-        <h3>Forum</h3>
-        <p>Have the time to share your thoughts with experts for each topic. 3</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_01.png" alt="Program 4">
-        <h3>Workshop</h3>
-        <p>Try brewing your own wine using traditional methods.</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_04.png" alt="Program 5">
-        <h3>CC Ignite</h3>
-        <p>Get opportunitiy to sniff and sip wine on a blindfold for a price.</p>
-      </div>
-    </div>
-  </div>
+      <nav class="px-[10%] md:flex h-[55px] justify-between shadow-xl">
+        <i class="menulist bi bi-list md:hidden p-4 text-[40px] pl-0"></i>
+        <a class="p-3 hidden md:flex" href="index.html"
+          ><img class="logo" src="assets/wine-glass.png" alt="" />wine~up</a
+        >
+        <ul class="items md:flex items-center justify-center">
+          <li class="close"><i class="bi bi-x-lg"></i></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="about.html">About</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Program</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">Join</a></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Sponsor</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">News</a></li>
+          <li
+            class="item p-1 m-1 hover:text-[#ec5242] border-solid border-4 border-[#ec5242]"
+          >
+            <a href="">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section
+        class="bg-text-clip h-screen bg-[url('assets/desk.png')] bg-contain bg-opacity-10"
+      >
+        <div
+          class="px-[10%] bg-[#5f5f5f50] h-screen flex justify-center flex-col gap-5"
+        >
+          <div class="text-[#ec5242]">
+            <h2 class="text-[25px] md:text-[50px]">"Hello! Sharing world"</h2>
+            <h1 class="text-[40px] md:text-[80px] (lato black)">
+              The Grape Escape<br />Wine Tasting and Braai
+            </h1>
+          </div>
+          <div
+            class="bg-[#d3d3d3] text-[#ec5242] w-[100%] md:w-[65%] px-[5%] py-[1%] border-solid border-2 border-[fff]"
+          >
+            <p>
+              Come raise your glass with us on yet another one of our annual
+              parties where we share inside information with future Scientists
+              on how to brew the finest wine in the world,
+            </p>
+          </div>
+          <div class="text-[#272a31]">
+            <h2 class="text-[25px] md:text-[40px] text-[#272a31]">
+              2023.March.17(Fri)~19(Sun)
+            </h2>
+            <p>@Stellenbosch Wine farms, Cape Town</p>
+          </div>
+        </div>
+      </section>
```

Comment:
Kindly use the proper layover color for this section. You have used `grey` in your application, however in the original design, it is `white`. Also, kindly either use the background image for the heading or use a similar color as in the provided design. I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.